app/src/main/java/net/kwatts/powtools/OWDevice.java

```diff
@@ -32,7 +32,7 @@
 public class OWDevice extends BaseObservable implements DeviceInterface {
     private static final String TAG = "OWTOOLS";
     private static final String NAME = "ONEWHEEL";
-    public static final SimpleDateFormat SIMPLE_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ", Locale.US);
+    public static final SimpleDateFormat SIMPLE_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ", Locale.US);
```

Comment:
FYI I just realize this breaks old files being read on my branch, I'll fix on the read.